libproxy/cmake/modules/pacrunner_duktape.cmk

```diff
@@ -0,0 +1,16 @@
+if(WIN32)
+  set(DUKTAPE_FOUND 0)
+else()
+  set(DUKTAPE_INCLUDE_DIR "WEBKIT_INCLUDE_DIR-NOTFOUND" CACHE PATH "Path to Duktape headers")
+  set(DUKTAPE_LIBRARIES   "WEBKIT_LIBRARIES-NOTFOUND"   CACHE FILEPATH "Path to Duktape libraries")
+
+  if(DUKTAPE_INCLUDE_DIR AND DUKTAPE_LIBRARIES)
+    message("Duktape headers: " ${DUKTAPE_INCLUDE_DIR})
+    message("Duktape libraries: " ${DUKTAPE_LIBRARIES})
+    set(DUKTAPE_FOUND 1)
+    math(EXPR NUM_PACRUNNER "${NUM_PACRUNNER}+1")
+    include_directories(${DUKTAPE_INCLUDE_DIR})
+    link_directories(${DUKTAPE_LIBRARIES})
+	link_libraries(duktape)
+  endif()
```

Comment:
Doesn't do `set(DUKTAPE_FOUND 0)` if not found

Reply:
Reverted the change to add a `set(DUKTAPE_FOUND 0)` at the beginning as it seems to break `px_check_modules` even when it comes after the set. `DUKTAPE_FOUND` is still 0 if the library is discovered.